LEAF_Nexus/sources/Group.php

```diff
@@ -162,9 +165,13 @@ public function addNew($groupTitle, $parentGroupID = 0)
      */
     public function deleteGroup($groupID)
     {
-    	if($groupID <= 10) {
-    		throw new Exception('Cannot delete system groups.');
-    	}
+        if(!is_numeric($groupID)){
+            throw new Exception('invalid group');
+            return 1;
```

Comment:
This should return `0` indicating failure, since it returns `1` when it is successful.